README.md

```diff
@@ -1,128 +1,233 @@
-# IBM Cloud <module_name> - Terraform Module
+# IBM Cloud Satellite location on GCP
 
-Mention the purpose of writing these modules.
+Use this terrafrom automation to set up satellite location on IBM cloud with google cloud host.
+It will provision satellite location and create 6 gcp host and assign 3 host to control plane, and provision ROKS satellite cluster and auto assign 3 host to cluster,
+Configure cluster worker pool to an existing ROKS satellite cluster.
 
-E.g:
+This is a collection of modules that make it easier to provision a satellite on IBM Cloud.
```

Comment:
to provision satellite location on GCP ... please correct

Reply:
Updated to GCP